.gitattributes

```diff
@@ -0,0 +1,57 @@
+# Treat all text files as text, with automatic line-ending handling
+* text=auto
```

Comment:
`auto` does weird things in Windows environments. Especially in spec-related files where LF is needed instead of CR/LF. Most text editors now on windows are just fine with LF.

If you are going to force a format, it is better, in my opinion, to set it explicitly to `lf`.

``
`*.txt eol=lf
*.gitignore text eol=lf
*.sh text eol=lf
*.md text eol=lf
*.mod text eol=lf
*.sum text eol=lf
*.go text eol=lf
*.yml text eol=lf
*.proto text eol=lf
*.json text eol=lf
*.html text eol=lf
*.svg text eol=lf
*.js text eol=lf
*.css text eol=lf
*.yaml text eol=lf
*.http text eol=lf
*.ps1 text eol=lf
*.g4 text eol=lf
*.interp text eol=lf
*.pem text eol=lf
*.cnf text eol=lf
*.conf text eol=lf
*.gitmodules text eol=lf
*.variants text eol=lf
*.cmake text eol=lf
*.bat text eol=lf
*.env text eol=lf
*.service text eol=lf
*.tmpl text eol=lf
*.partial text eol=lf
*.liquid text eol=lf
*.tokens text eol=lf
*.attr text eol=lf
*.in text eol=lf
*.h text eol=lf
*.c text eol=lf
*.kts text eol=lf
*.properties text eol=lf
*.rst text eol=lf
*.gradle text eol=lf
*.java text eol=lf
*.kt text eol=lf
```